corehq/apps/export/export.py

```diff
@@ -375,6 +375,12 @@ def write_export_instance(writer, export_instance, documents,
                 if ALL_CASE_TYPE_TABLE in table.path and doc['type'] not in path_names:
                     continue
 
+                # For geojson exports it is important to reliably know whether
+                # the split_multiselects setting is enabled at the point when the
+                # table's rows are handled
+                if writer.format == "geojson":
+                    table.split_multiselects = export_instance.split_multiselects
```

Comment:
This will, for the duration of the routine.

> Anyhow, can't we get split_multiselects from the export instance itself wherever needed?

Unfortunately not. Although, now that I reviewed the code I did get an idea I how I might not have to "hack on" the value. 

I'll explore and update...